extensions/support-page/main.ts

```diff
@@ -1,13 +1,13 @@
 import { LensMainExtension, windowManager } from "@k8slens/extensions";
-import { supportPageURL } from "./src/support.route";
+import { pageUrl } from "./src/common-vars";
 
 export default class SupportPageMainExtension extends LensMainExtension {
   appMenus = [
     {
       parentId: "help",
       label: "Support",
-      click() {
-        windowManager.navigate(supportPageURL());
+      click: () => {
+        windowManager.navigate(this.getPageUrl(pageUrl)); // todo: simplify
```

Comment:
Did you see my comment [here](https://github.com/lensapp/lens/pull/1323#discussion_r521352669)? Maybe this is similar to what you are already planning...

Reply:
1. Page registration doesn't have ID
1. Registered page exists only in `LensRendererExtension` but access to URL might need to be from `LensMainExtension` as well (like from top-menu support item)
1. If even this possible should be done in another PR